### PR TITLE
fix: correct meeting link in README.md

### PR DESCRIPTION
**Pull request by AleJo2995**
Correct meeting link

### CODE REVIEW

File: community/working-groups/compliance/README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Reviewing industry and governmental standards (e.g., NIST, PCI, HIPAA) from a cl
 ## Meeting Information
 
 - **Weekly Meetings:** 10:00 AM Eastern Time (US and Canada)
-- **Meeting Link:** [Zoom Meeting](https://zoom.us/j/92729235315?pwd=ZFIxU3RSanlVODh4a1g2SFdJOGpoZz09)
+- **Meeting Link:** [Zoom Meeting]([https://zoom.us/j/92729235315?pwd=ZFIxU3RSanlVODh4a1g2SFdJOGpoZz09](https://zoom-lfx.platform.linuxfoundation.org/meeting/94852354733?password=c99601ab-0a5a-4ea9-98e3-af9d12c59547))
```
Comment (brandtkeller) [inline community/working-groups/compliance/README.md:29]: Looks like the link was nested - perhaps you meant:

```suggestion
- **Meeting Link:** [Zoom Meeting](https://zoom-lfx.platform.linuxfoundation.org/meeting/94852354733?password=c99601ab-0a5a-4ea9-98e3-af9d12c59547)
```

**Review by brandtkeller — CHANGES_REQUESTED** [review]

File: community/working-groups/compliance/README.md (hunk 1 above)
Comment (AleJo2995) [reply community/working-groups/compliance/README.md:29]: You're completely right, thanks a lot

**Comment by AleJo2995** [comment]
@brandtkeller just FYI I am holding this to confirm with Anca Sailer if we want to add more changes to this page so it gets up to date. Thanks

**Comment by AleJo2995** [comment]
@brandtkeller I have confirmed with Anca that this is the only information we're going to include this time. I am seeing the linters and they seem to be not associated with the changes themselves, any insight here?

**Review by brandtkeller — APPROVED** [review]
Confirmed valid link - lint failures are from existing errors that can be fixed iteratively.

**Pull request merged**